packages/devtools_app/lib/src/timeline/flutter/timeline_flame_chart.dart

```diff
@@ -264,25 +264,30 @@ class _FullTimelineFlameChartState
       if (row >= rows.length) {
         rows.addAll(List.generate(
           row - (rows.length - 1) + _rowOffsetForBottomTimeline,
-          (i) => FlameChartRow(nodes: [], index: i),
+          (_) => FlameChartRow(nodes: []),
         ));
       }
     }
 
+    double leftForEvent(TimelineEvent event) =>
```

Comment:
dartfmt makes these methods look really ugly. Consider using ` {}` instead of `=>` for clarity